semgrep-core/tests/js/dot_access_ellipsis.js

```diff
@@ -0,0 +1,14 @@
+// MATCH:
```

Comment:
there's some kinds of convention on how to name those tests (you can see generate_cheatsheet.py for some hints).
but basically the beginning of the filename should repreresent a big feature of semgrep, e.g., dots_... metavar_... equivalence_... so here you probably want dots_field_chaining.js